src/test/java/features/birsen.feature

```diff
@@ -0,0 +1,5 @@
+Feature: A description
```

Comment:
Please add javadoc

Reply:
There is no accept in feature class javadoc